docs/usage/shoot_overlay_network.md

```diff
@@ -0,0 +1,77 @@
+# Enable / disable overlay network for shoots with Calico
+Gardener can be used with or without the overlay network. From now on, the default configuration of shoot clusters is without overlay network.
```

Comment:
It is not clear for the reader what is "From now on". I would rather suggest you to put concrete component versions. For example starting `provider-aws@vX.Y.Z` for AWS Shoots.
And this defaulting was recently reverted as part of https://github.com/gardener/gardener-extension-provider-aws/issues/621 - the default currently is "with overlay" until the colleagues rework the defaulting part.